packages/jss-plugin-rule-value-function/src/index.js

```diff
@@ -16,6 +17,34 @@ const fnRuleNs = `fnStyle${++now}`
 
 type StyleRuleWithRuleFunction = StyleRule & {[key: string]: Function}
 
+function getFunctionStyles(styles: Object): Object | null {
+  let to = null
+
+  for (const key in styles) {
+    const value = styles[key]
+    const type = typeof value
+
+    if (isObservable(value)) {
+      continue
+    }
+
+    if (type === 'function') {
+      if (!to) to = {}
+      to[key] = value
+    } else if (type === 'object' && value !== null && !Array.isArray(value)) {
```

Comment:
Since I just excluded null and undefined in the previous comment, we can remove `value !== null` check